test/development/acceptance-app/rsc-build-errors.test.ts

```diff
@@ -184,7 +184,11 @@ describe('Error overlay - RSC build errors', () => {
          4 |   return 'client-only-lib'
 
        You're importing a component that imports client-only. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by default.
-       Learn more: https://nextjs.org/docs/app/building-your-application/rendering"
+       Learn more: https://nextjs.org/docs/app/building-your-application/rendering
+
+       Example import trace:
+         ./app/server-with-errors/client-only-in-server/client-only-lib.js [app-rsc]
+         ./app/server-with-errors/client-only-in-server/page.js [app-rsc] [entrypoint]"
```

Comment:
```suggestion
       Example import trace (app-rsc):
         ./app/server-with-errors/client-only-in-server/client-only-lib.js
         ./app/server-with-errors/client-only-in-server/page.js"
```

For a single import trace, put the layer into the title to avoid repeating it.

Reply:
did a version of this